assignments/002-unix-intro/unix-intro.sh

```diff
@@ -0,0 +1,553 @@
+
+[hb: lonely-fastqs]--% ls -R /hb/home/ccolumbu/projects/con-gen-csu/assignments/002-unix-intro/lonely-fastqs/
+/hb/home/ccolumbu/projects/con-gen-csu/assignments/002-unix-intro/lonely-fastqs/:
+R1/  R2/
+
+/hb/home/ccolumbu/projects/con-gen-csu/assignments/002-unix-intro/lonely-fastqs/R1:
+DPCh_plate1_A05_S5.R1.fq.gz   DPCh_plate1_B12_S24.R1.fq.gz  DPCh_plate1_D11_S47.R1.fq.gz  DPCh_plate1_F06_S66.R1.fq.gz	DPCh_plate1_H05_S89.R1.fq.gz
+DPCh_plate1_A06_S6.R1.fq.gz   DPCh_plate1_C05_S29.R1.fq.gz  DPCh_plate1_D12_S48.R1.fq.gz  DPCh_plate1_F11_S71.R1.fq.gz	DPCh_plate1_H06_S90.R1.fq.gz
+DPCh_plate1_A11_S11.R1.fq.gz  DPCh_plate1_C06_S30.R1.fq.gz  DPCh_plate1_E05_S53.R1.fq.gz  DPCh_plate1_F12_S72.R1.fq.gz	DPCh_plate1_H11_S95.R1.fq.gz
+DPCh_plate1_A12_S12.R1.fq.gz  DPCh_plate1_C11_S35.R1.fq.gz  DPCh_plate1_E06_S54.R1.fq.gz  DPCh_plate1_G05_S77.R1.fq.gz	DPCh_plate1_H12_S96.R1.fq.gz
+DPCh_plate1_B05_S17.R1.fq.gz  DPCh_plate1_C12_S36.R1.fq.gz  DPCh_plate1_E11_S59.R1.fq.gz  DPCh_plate1_G06_S78.R1.fq.gz
+DPCh_plate1_B06_S18.R1.fq.gz  DPCh_plate1_D05_S41.R1.fq.gz  DPCh_plate1_E12_S60.R1.fq.gz  DPCh_plate1_G11_S83.R1.fq.gz
+DPCh_plate1_B11_S23.R1.fq.gz  DPCh_plate1_D06_S42.R1.fq.gz  DPCh_plate1_F05_S65.R1.fq.gz  DPCh_plate1_G12_S84.R1.fq.gz
+
+/hb/home/ccolumbu/projects/con-gen-csu/assignments/002-unix-intro/lonely-fastqs/R2:
+DPCh_plate1_A05_S5.R2.fq.gz   DPCh_plate1_B12_S24.R2.fq.gz  DPCh_plate1_D11_S47.R2.fq.gz  DPCh_plate1_F06_S66.R2.fq.gz	DPCh_plate1_H05_S89.R2.fq.gz
+DPCh_plate1_A06_S6.R2.fq.gz   DPCh_plate1_C05_S29.R2.fq.gz  DPCh_plate1_D12_S48.R2.fq.gz  DPCh_plate1_F11_S71.R2.fq.gz	DPCh_plate1_H06_S90.R2.fq.gz
+DPCh_plate1_A11_S11.R2.fq.gz  DPCh_plate1_C06_S30.R2.fq.gz  DPCh_plate1_E05_S53.R2.fq.gz  DPCh_plate1_F12_S72.R2.fq.gz	DPCh_plate1_H11_S95.R2.fq.gz
+DPCh_plate1_A12_S12.R2.fq.gz  DPCh_plate1_C11_S35.R2.fq.gz  DPCh_plate1_E06_S54.R2.fq.gz  DPCh_plate1_G05_S77.R2.fq.gz	DPCh_plate1_H12_S96.R2.fq.gz
+DPCh_plate1_B05_S17.R2.fq.gz  DPCh_plate1_C12_S36.R2.fq.gz  DPCh_plate1_E11_S59.R2.fq.gz  DPCh_plate1_G06_S78.R2.fq.gz
+DPCh_plate1_B06_S18.R2.fq.gz  DPCh_plate1_D05_S41.R2.fq.gz  DPCh_plate1_E12_S60.R2.fq.gz  DPCh_plate1_G11_S83.R2.fq.gz
+DPCh_plate1_B11_S23.R2.fq.gz  DPCh_plate1_D06_S42.R2.fq.gz  DPCh_plate1_F05_S65.R2.fq.gz  DPCh_plate1_G12_S84.R2.fq.gz
+
+#### Ex. 12: Use the recursive (-r) option to remove `lonely-fastqs`
+####         and its contents
+
+[hb: 002-unix-intro]--% rm -r lonely-fastqs/
+
+#### Ex. 13: Make two new directories in the top level of the
+####         repo called `fq_AB` and `fq_notAB`
+
+[hb: con-gen-csu]--% mkdir fq_AB fq_notAB
+
+#### Ex. 14: Using the `[` `]` copy the R1 files in `fastqs` that
+####         belong to sample names (the part immediately after `DPCh_plate1_`)
+####         starting with an `A` or `B` to `fq_AB`.
+
+[hb: con-gen-csu]--% cp assignments/002-unix-intro/fastq/*DPCh_plate1_[AB]*R1.fq.gz /hb/home/ccolumbu/projects/con-gen-csu/fq_AB
```

Comment:
You don't have to give it the absolute path like:
`/hb/home/ccolumbu/projects/con-gen-csu/fq_AB` here.  You could just use the relative path: `fq_AB`